syntax = "proto3";
package m2g.pb;

enum SpriteType {
  NO_SPRITE = 0;
}
